lib/nunjucks/govuk-prototype-kit/layouts/unbranded.html

```diff
@@ -4,7 +4,7 @@
   <link rel="shortcut icon" href="/extension-assets/govuk-prototype-kit/lib/assets/images/unbranded.ico" type="image/x-icon" />
 {% endblock %}
 
-{% block head %}
+{% block stylesheets %}
```

Comment:
should unbranded have a meta block too, same approach as branded?

Reply:
unbranded extends govuk-branded, and doesn't override the head block, just the stylesheets block so the meta block propagates through.

Reply:
oh of course thanks